public/opensearch.xml

```diff
@@ -0,0 +1,6 @@
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/" xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+  <ShortName>IPFS Search</ShortName>
+  <Description>IPFS Search</Description>
+  <InputEncoding>UTF-8</InputEncoding>
+  <Url type="text/html" template="http://localhost:8080/#/search?q={searchTerms}" />
```

Comment:
We're not running on localhost, we're running (and expect users to request our engine on `ipfs-search.com`. Which also happens to be our short name. I would also suggest that our description is actually meaningful, e.g. our tagline 'Search the Distributed Web'.

It would also be great to have a `startPage` parameter there.